find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
```

Comment:
This comment is helpful, but you might consider (even if this is a small change) getting into the practice of adding a comment to your PL discussing what you changed/added, how it improves/impacts your code base -- for reviewer folx to be able to better understand your code's goals and scope quickly